crates/spl_network_messages/src/lib.rs

```diff
@@ -34,10 +35,17 @@ impl Default for HulkMessage {
 pub struct StrikerMessage {
     pub player_number: PlayerNumber,
     pub pose: Pose2<Field>,
-    pub ball_position: Option<BallPosition<Field>>,
+    pub ball_position: BallPosition<Field>,
+    // TODO: make non-optional
```

Comment:
What is blocking you from that? Is this something for a next iteration?

Reply:
Is done